code/src/components/Input.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const Input = ({nameValue, onInputNameChange}) => {
+    return(
+        <div className="formelement">
+            <label htmlFor={nameValue}></label>
```

Comment:
There is no content in this label, which also gives an error in the accessibility check in WAVE. Could be good to update to have some content going in there :)